vms/platformvm/state/state.go

```diff
@@ -805,6 +807,29 @@ func (s *state) AddTx(tx *txs.Tx, status status.Status) {
 	}
 }
 
+func (s *state) GetRewardConfig(subnetID ids.ID) (reward.Config, error) {
+	primaryNetworkCfg := s.cfg.RewardConfig
+	if subnetID == constants.PrimaryNetworkID {
+		return primaryNetworkCfg, nil
+	}
+
+	transformSubnetIntf, err := s.GetSubnetTransformation(subnetID)
+	if err != nil {
+		return reward.Config{}, err
+	}
```

Comment:
this is probably wrong. We should special case missing transgormation error, since the transgormation may be in some higher up diffs